QuickBytes/src/main/java/com/quickbytes/repository/RequestComboRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.quickbytes.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+import com.quickbytes.model.RequestCombo;
+
+public interface RequestComboRepository extends JpaRepository<RequestCombo,Long>{
+	
+	@Query("select rc from RequestCombo rc where rc.request.requestId=?1")
+	List<RequestCombo> getAllRequestCombosByRequestId(Long oid);
```

Comment:
Haven't tried to see if it matters but the request path for that is "rid"

Reply:
Doesn't matter but definitely missed this when replacing oid with rid, ty!